susie/src/main/resources/makeSuSiE.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/python3
+from optparse import OptionParser
+import pandas as pd
+import numpy as np
+import shutil
+import subprocess
+import os
+
+s3_in=os.environ['INPUT_PATH']
+s3_out=os.environ['OUTPUT_PATH']
+
+# def finds json files in the directory
+def make_json_files(directory):
+	subprocess.check_call(['aws', 's3', 'cp', directory, 'input/', '--recursive'])
+	subprocess.run('cat input/*.json > input.json', shell=True)
+	shutil.rmtree('input')
+
+def make_ld_files(directory):
+	subprocess.check_call(['aws', 's3', 'cp', directory, 'input/', '--recursive'])
+	subprocess.run('cat input/*.ld > snp_ld.ld', shell=True)
+	shutil.rmtree('input')
+
+def main():
+	usage = "usage: %prog [options]"
+	parser = OptionParser(usage)
+	parser.add_option("","--phenotype", default=None)
+	parser.add_option("","--ancestry", default=None)
+
+	(options, args) = parser.parse_args()
```

Comment:
just args = parser.parse_args() and use args instead of options

Reply:
done!